cli/cmd/vuln_host.go

```diff
@@ -1018,3 +946,119 @@ func hostScanPackagesVulnPackagesView(vulns []api.HostScanPackageVulnDetails) []
 
 	return out
 }
+
+// Build the cli output for vuln host show-assessment
+func buildVulnHostReports(assessment api.HostVulnHostAssessment) error {
+	mainReport := hostVulnHostDetailsMainReportTable(assessment)
+	filteredCves, filtered := hostVulnCVEs(assessment.CVEs)
+	assessment.CVEs = filteredCves
+
+	detailsReport := buildVulnHostsDetailsTable(filteredCves)
+
+	if cli.JSONOutput() {
+		if err := cli.OutputJSON(assessment); err != nil {
+			return err
+		}
+		return nil
+	} else {
+		cli.OutputHuman(mainReport)
+		cli.OutputHuman(detailsReport)
+		if filtered != "" {
+			cli.OutputHuman(filtered)
+		}
+		return nil
+	}
+}
+
+// Build the cli output for vuln host list-cves
+func buildListCVEReports(cves []api.HostVulnCVE) error {
+	filteredCves, filtered := hostVulnCVEs(cves)
+
+	if cli.JSONOutput() {
+		if filteredCves == nil {
+			if err := cli.OutputJSON(buildHostVulnCVEsToTableError()); err != nil {
+				return err
+			}
+		} else {
+			if err := cli.OutputJSON(filteredCves); err != nil {
+				return err
+			}
+		}
+		return nil
+	}
+
+	if len(cves) == 0 {
+		// @afiune add a helpful message, possible things are:
+		// 1) host vuln feature is not enabled on the account
+		// 2) user doesn't have agents deployed
+		// 3) there are actually NO vulnerabilities on any host
+		cli.OutputHuman("There are no vulnerabilities on any host in your environment.\n")
+		return nil
+	}
+
+	if vulCmdState.Packages {
+		packages, filtered := hostVulnPackagesTable(cves, true)
+		cli.OutputHuman(
+			renderSimpleTable(
+				[]string{"CVE Count", "Severity", "Package", "Current Version", "Fix Version", "Pkg Status", "Hosts"},
+				packages,
+			),
+		)
+		if filtered != "" {
+			cli.OutputHuman(filtered)
+		}
+		return nil
+	}
+
+	rows := hostVulnCVEsTable(filteredCves)
+	// if the user wants to show only online or
+	// offline hosts, show a friendly message
+	if len(rows) == 0 {
+		cli.OutputHuman(buildHostVulnCVEsToTableError())
+		return nil
+	}
+
+	cli.OutputHuman(
+		renderSimpleTable(
+			[]string{"CVE ID", "Severity", "Score", "Package", "Current Version",
+				"Fix Version", "OS Version", "Hosts", "Pkg Status", "Vuln Status"},
+			rows,
+		),
+	)
+
+	if filtered != "" {
+		cli.OutputHuman(filtered)
+	}
+
+	if !vulCmdState.Active {
+		cli.OutputHuman(
+			"\nTry adding '--active' to only show vulnerabilities of packages actively running.\n",
+		)
+	} else if !vulCmdState.Fixable {
+		cli.OutputHuman(
+			"\nTry adding '--fixable' to only show fixable vulnerabilities.\n",
+		)
+	}
+	return nil
+}
+
+// Build the cli output for vuln host scan-package-manifest
+func buildVulnHostScanPkgManifestReports(response api.HostVulnScanPkgManifestResponse) error {
+	if len(response.Vulns) == 0 {
+		// @afiune add a helpful message, possible things are:
+		cli.OutputHuman("There are no vulnerabilities found.\n")
```

Comment:
I wanted to change this for months, let us do it now that I remembered and we can:
```suggestion
		cli.OutputHuman(fmt.Sprintf("There are no vulnerabilities found! Time for %s\n", randomEmoji()))
```